bindings/python/src/decoders.rs

```diff
@@ -161,8 +161,8 @@ impl PyByteLevelDec {
 ///     cleanup (:obj:`bool`, `optional`, defaults to :obj:`True`):
 ///         Whether to cleanup some tokenization artifacts. Mainly spaces before punctuation,
 ///         and some abbreviated english forms.
-#[pyclass(extends=PyDecoder, module = "tokenizers.decoders", name=WordPiece)]
-#[text_signature = "(self, prefix=\"##\", cleanup=True)"]
+#[pyclass(extends=PyDecoder, module = "tokenizers.decoders", name = "WordPiece")]
+#[pyo3(text_signature = "(self, prefix=\"##\", cleanup=True)")]
```

Comment:
@messense FYI in https://github.com/PyO3/pyo3/pull/2234 (I guess PyO3 0.16.3) the text signature can also be part of `#[pyclass()]` arguments.

Reply:
pyo3 0.16.3 isn't released yet anyway.

Reply:
> pyo3 0.16.3 isn't released yet anyway.

Not sure what the timeline is now for tokenizers 0.13.0 (just saw 0.12.1 getting prepared), but if it's worth doing the text signature bits before that, I wanted to add that pyo3 0.16.3 has now been [released](https://github.com/PyO3/pyo3/releases/tag/v0.16.3) a few days ago.